src/java.base/share/classes/java/lang/invoke/InnerClassLambdaMetafactory.java

```diff
@@ -362,36 +361,53 @@ else if (accidentallySerializable)
         // Define the generated class in this VM.
 
         final byte[] classBytes = cw.toByteArray();
-        // If requested, dump out to a file for debugging purposes
-        if (dumper != null) {
-            AccessController.doPrivileged(new PrivilegedAction<>() {
-                @Override
-                public Void run() {
-                    dumper.dumpClass(lambdaClassName, classBytes);
-                    return null;
-                }
-            }, null,
-            new FilePermission("<<ALL FILES>>", "read, write"),
-            // createDirectories may need it
-            new PropertyPermission("user.dir", "read"));
-        }
         try {
             // this class is linked at the indy callsite; so define a hidden nestmate
             Lookup lookup;
-            if (useImplMethodHandle) {
-                lookup = caller.defineHiddenClassWithClassData(classBytes, implementation, !disableEagerInitialization,
-                                                               NESTMATE, STRONG);
-            } else {
-                lookup = caller.defineHiddenClass(classBytes, !disableEagerInitialization, NESTMATE, STRONG);
+            try {
+                if (useImplMethodHandle) {
+                    lookup = caller.defineHiddenClassWithClassData(classBytes, implementation, !disableEagerInitialization,
+                        NESTMATE, STRONG);
```

Comment:
nit: formatting - align this line to the first argument `classBytes` in L369.

Reply:
Done.